README.md

```diff
@@ -31,11 +50,14 @@ samples, guidance on mobile development, and a full API reference.
   ```
   `--dart-define` is used to pass environment configuration.
 
-#### Environment variables
+#### ⚙️ Environment variables
 
-- `BEDROCK_API`: This should be pointed to the base url of where the API is running.
+| Variable              | Default Value          | Description |
+| -                     | -                      | -           |
+| `BEDROCK_API`         | https://localhost:2300 | Base url of where the API is running |
```

Comment:
I think this is good, but I think we should have some sort of facility inside the base app to switch between development, staging and production. It is quite common that we give out the app to early users where we want them to explicitly use one of those environments. 

We can do this in a different PR set